docs: update future work item on voting

nielsleadholm opened the pull request:
This updates the description of the Future Work item on voting with more detail re. concrete next steps. This information is informed by the scoping and discussion in this PR: https://github.com/thousandbrainsproject/tbp.monty/pull/358

vkakerbeck commented:
Sorry for the confusion. I think when we talked about this, I understood it as "what's the best first experiment to test an implementation of this?" instead of "what are all the experiments that would need to be run to test this?". I agree that the 1 object scenario would be the best first step to test a prototype of this and implement the basic infrastructure for this approach. However, I think it would be worth clarifying that this isn't the only experiment. I'm not sure I would integrate a prototype of this into Monty that doesn't also demonstrate it working in multi-object scenarios, where the neighboring objects vary between episodes. Does that make sense?

nielsleadholm commented:
Sorry for letting this one go stale, I've updated the final description to make it clearer that replicating performance on the existing benchmark would only be a first step. 

I've also updated the description to clarify that the existing RFC would need re-writing.
